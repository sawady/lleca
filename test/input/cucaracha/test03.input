
fun main() {
  putNum(123456)
  putChar(10)
  putNum(654321)
  putChar(10)
}
