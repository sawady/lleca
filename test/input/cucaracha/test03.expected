Cons(
  Function(
    main,
    Nil,
    Unit,
    Cons(
      StmtCall(
        putNum,
        Cons(
          ExprConstNum(
            123456
          ),
          Nil
        )
      ),
      Cons(
        StmtCall(
          putChar,
          Cons(
            ExprConstNum(
              10
            ),
            Nil
          )
        ),
        Cons(
          StmtCall(
            putNum,
            Cons(
              ExprConstNum(
                654321
              ),
              Nil
            )
          ),
          Cons(
            StmtCall(
              putChar,
              Cons(
                ExprConstNum(
                  10
                ),
                Nil
              )
            ),
            Nil
          )
        )
      )
    )
  ),
  Nil
)
